### PR TITLE
[Fix][UI] style change warningInstance wechat keyword translate

### PR DESCRIPTION
**Pull request by chengchuen**
style change warningInstance wechat keyword translate i18n

### CODE REVIEW

File: dolphinscheduler-dao/pom.xml [hunk 1]
```diff
@@ -63,6 +63,11 @@
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
         </dependency>
+        <dependency>
```
Comment (lenboo) [inline dolphinscheduler-dao/pom.xml:81]: Is it unnecessary change?

**Comment by songjianet** [comment]
Thank you for your contribution, but since this issue has not been dealt with for a long time, I will close it. If you still have problems in the future, you can re-submit a new issue.
